projects/cdk/schematics/ng-update/constants/warnings.ts

```diff
@@ -106,4 +106,10 @@ export const MIGRATION_WARNINGS: readonly MigrationWarning[] = [
         moduleSpecifier: '@taiga-ui/core',
         message: 'colorFallback has been deleted in 3.0, please use CSS variables',
     },
+    {
```

Comment:
we can add this to main branch

Reply:
Hmm.. It's breaking change, we can't
if you mean migration only, that doesn't make sense in main branch

Reply:
oh, sorry